src/neo4j_genai/retrievers/external/weaviate/examples/text_search_local_embedder.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from neo4j_genai.retrievers.external.weaviate import WeaviateNeo4jRetriever
+from langchain_community.embeddings import HuggingFaceEmbeddings
+from neo4j import GraphDatabase
+import weaviate
+
+NEO4J_URL = "neo4j://localhost:7687"
+NEO4J_AUTH = ("neo4j", "password")
+
+
+def main():
+    with GraphDatabase.driver(NEO4J_URL, auth=NEO4J_AUTH) as neo4j_driver:
+        with weaviate.connect_to_local() as w_client:
+            embedder = HuggingFaceEmbeddings(model_name="all-MiniLM-L6-v2")
```

Comment:
I see starting the container with `all-MiniLM-L6-v2` takes the most time in the E2E test. Maybe consider using `paraphrase-albert-small-v2` model instead as this is the smallest sentence-transformers model?

Reply:
Based on https://www.sbert.net/docs/pretrained_models.html

Reply:
I think we're good with the current one, if you're ok with that. Locally it's only slow the first time.